### DISCUSSION TITLE
Help with Cosmos REST API

### DISCUSSION THREAD

**Discussion by eilon-lotem** [cosmos/community#66] [Clients (JS)] [answered]
Hi,

I need help with Cosmos Hub REST API,

There is an event type /dydxprotocol.bridge.MsgAcknowledgeBridges

[https://dydx-rest.kingnodes.com/cosmos/tx/v1beta1/txs?limit=50&&orderBy=1&events=message.action%3D'/dydxprotocol.bridge.MsgAcknowledgeBridges'](https://dydx-rest.kingnodes.com/cosmos/tx/v1beta1/txs?limit=50&&orderBy=1&events=message.action%3D%27/dydxprotocol.bridge.MsgAcknowledgeBridges%27)

What do I need to do to filter only by specific address,

For example: dydx18qm63sare7ud2p9qnwm9a4w0ck44w7pf3uhrgx

**Comment by tac0turtle** [comment, marked as answer]
you can use the address parameter to filter by address
